mlir/lib/ExecutionEngine/ROCm/rocm-runtime-wrappers.cpp

```diff
@@ -31,6 +31,18 @@ int32_t reportErrorIfAny(hipError_t result, const char *where) {
 }
 } // anonymous namespace
 
+typedef union bf16_fp32_cvt {
+  uint u32;
+  unsigned short ushortvec[2];
+  float f32;
+} bf16_fp32_cvt_t;
+
+static float bfloat16_to_float(ushort src_val) {
+  bf16_fp32_cvt_t target_val;
+  target_val.ushortvec[1] = src_val;
```

Comment:
need to assign lower bits (ushortvec[0]).

Reply:
assigned

Reply:
added it